src/Game/Menu.java

```diff
@@ -85,14 +87,20 @@ public static void resize() {
   public static boolean switchList(Object menu) {
     if (menu == spell_menu) {
       try {
-        int[] scroll = (int[]) Reflection.menuScroll.get(spell_menu);
-
-        // Swap scroll values
-        spell_swap_scroll[spell_swap_idx] = scroll[spell_handle];
-        spell_swap_idx ^= 1;
-        scroll[spell_handle] = spell_swap_scroll[spell_swap_idx];
-
-        Reflection.menuScroll.set(spell_menu, scroll);
+    	  int[] scroll = (int[]) Reflection.menuScroll.get(spell_menu);
+    	  
+    	  if (Settings.KEEP_MAGIC_PRAYER_TAB_CURSOR.get(Settings.currentProfile)
+     			 && !Settings.SPEEDRUNNER_MODE_ACTIVE.get(Settings.currentProfile)) {
```

Comment:
Speedrunner mode explanation will need to say that this was added to it

Reply:
placed explanation